cfgov/jinja2/v1/_includes/blocks/raf_tool.html

```diff
@@ -1,19 +1,6 @@
-<style>
-  #rental-assistance-finder {
-    max-width: 41.875rem;
-  }
-  #rental-assistance-finder dl a {
-    word-break: break-all;
-  }
-  #rental-assistance-finder .m-form-field + .m-form-field {
-    margin-top: 15px
-  }
-  .no-js .js-is-hidden {
-    display: block !important;
-  }
-</style>
+<link rel="stylesheet" href="{{ static('apps/erap/css/main.css') }}">
```

Comment:
I moved this to the app assets so the JS and CSS were in one place, but it could be inlined as it was if that is preferred.

Reply:
Ideally we'd be able to load the tool's CSS in `base.html` from a `Media` entry on the RAFT block (the way the JS is currently added to the page), but that would require some global work so I think this update makes sense now.